tools/Gen2Master/MoveFromGeneration2Master.ps1

```diff
@@ -361,11 +361,11 @@ Function Update-MappingJson {
         $MappingPath = Join-Path -Path (Join-Path -Path $PSScriptRoot -ChildPath '..') -ChildPath "CreateMappings_rules.json"
         $MappingObject = Get-Content -Path $MappingPath | ConvertFrom-Json
         Foreach ($Item in $MappingObject) {
-            If ($ModuleName -eq $Item.regex) {
+            If ($ModuleName -eq $Item.module) {
```

Comment:
Do all generated modules have "module" property?

Reply:
Thanks. I've updated the condition so that a new rule will be added only if there are no rules match module OR regex.